src/content/nft/index.md

```diff
@@ -344,11 +344,11 @@ Importantly this improves the energy efficiency while preserving Ethereum's dece
 
 _We’ve provided the basic comparison to Visa to baseline your understanding of Eth2 energy consumption against a familiar name. However, in practice, it’s not really correct to compare based on number of transactions. Ethereum’s energy output is time-based. If Ethereum did more or less transactions from one minute to the next, the energy output would stay the same._
 
-_It’s also important to remember that Ethereum does more than just financial transactions, it’s a platform for applications, so a fairer comparison might be to many companies/industries including Visa, AWS and more!_
+_It’s also important to remember that Ethereum does more than just financial transactions, it’s a platform for applications, so a fairer comparison might be too many companies/industries including Visa, AWS and more!_
```

Comment:
Hey :-) This suggestion is incorrect.  

From https://www.grammarly.com/blog/to-too/

> To is a preposition with several meanings, including “toward” and “until.”
> Too is an adverb that can mean “excessively” or “also.”

```suggestion
_It’s also important to remember that Ethereum does more than just financial transactions, it’s a platform for applications, so a fairer comparison might be to many companies/industries including Visa, AWS and more!_
```